bundler.d/graphiql.rb

```diff
@@ -0,0 +1,3 @@
+group :development, :test do
```

Comment:
why is this needed in `test`?

Reply:
I guess it's not needed there. Development should be enough. Good catch.

Reply:
I removed it from `:test`

Reply:
I guess it should go into `development.rb` then, as it's in that bundler group.